Calibration/HcalAlCaRecoProducers/python/ALCARECOHcalCalIsoTrkFromAlCaRaw_Output_cff.py

```diff
@@ -0,0 +1,21 @@
+import FWCore.ParameterSet.Config as cms
+
+# output block for alcastream HCAL Isotrk
+# output module 
+#  module alcastreamHcalIsotrkOutput = PoolOutputModule
+OutALCARECOHcalCalIsoTrkFromAlCaRaw_noDrop = cms.PSet(
+    # use this in case of filter available
+    SelectEvents = cms.untracked.PSet(
+        SelectEvents = cms.vstring('pathALCARECOHcalCalIsoTrkFromAlCaRaw')
+        ),
+    outputCommands = cms.untracked.vstring( 
+        'keep *_alcaHcalIsotrkProducer_HcalIsoTrack_*',
+        'keep *_alcaHcalIsotrkProducer_HcalIsoTrackEvent_*',
+        'keep *_genParticles_*_*',
+        )
+)
+
+
+import copy
+OutALCARECOHcalCalIsoTrkFromAlCaRaw=copy.deepcopy(OutALCARECOHcalCalIsoTrkFromAlCaRaw_noDrop)
```

Comment:
can this be cloned instead of `copy`ed?

Reply:
Cloning seems to be fine (I found and followed an equivalent example from you that I found 🙂  ).